Web.Api/Controllers/ListController.cs

```diff
@@ -90,5 +91,39 @@ public Task<ActionResult<ListDto>> MoveTaskToList([FromHeader] Guid userId, Guid
         {
             throw new NotImplementedException();
         }
+
+        [HttpDelete("{listId}", Name = "DeleteList")]
+        public async Task<ActionResult<ListDto>> DeleteList([FromHeader] Guid listId, Guid userId)
+        {
+            if (!await _unitOfWork.User.IsUserInDbAsync(userId))
+            {
+                return StatusCode(403);
+            }
+
+            List? list = await _unitOfWork.List.GetListByIdAsync(listId, userId); 
+            if (list is null)
+            {
+                return NotFound();
```

Comment:
```suggestion
                return NotFound(listId);
```


Echo the not found id